libmscore/part.cpp

```diff
@@ -34,6 +36,7 @@ namespace Ms {
 Part::Part(Score* s)
     : ScoreElement(s)
 {
+    _id = QUuid::createUuid().toString();
```

Comment:
Maybe better use just a static variable?, it is better for debugging and performance

Reply:
fixed